readme.md

```diff
@@ -875,6 +875,10 @@ Plan how many assertion there are in the test. The test will fail if the actual
 
 End the test. Only works with `test.cb()`.
 
+###### `t.log(message)`
+
+Print a log message contextually alongside the test result instead of immediately streaming the message to stdout like `console.log`.
```

Comment:
Along the lines of https://nodejs.org/api/console.html#console_console_log_data_args, perhaps:

> Print a log message contextually alongside the test result instead of immediately printing it to `stdout` like `console.log`.